src/analyses/base.ml

```diff
@@ -908,6 +910,12 @@ struct
         | `Bot -> Queries.Result.bot q (* TODO: remove *)
         | _ -> Queries.Result.top q
       end
+    | Q.EvalThread e -> begin
+      match eval_rv (Analyses.ask_of_ctx ctx) ctx.global ctx.local e with
+        | `Thread a -> a
+        | `Bot -> Queries.Result.bot q (* TODO: remove *)
```

Comment:
When would this happen? For globals?

Reply:
I'm not sure, hopefully never. I just based it on all the other cases which historically had such case and in #221 I didn't want to remove them (let `Bot` return top query result instead) just in case it somehow matters.

If a thread ID is global, then it should get global initialized to a non-bottom anyway, so that shouldn't be it.